docs/UsageInWebAPI.md

```diff
@@ -152,3 +152,59 @@ Check [tutorial/notebook.ipynb](https://github.com/PyFPDF/fpdf2/blob/master/tuto
 Usage of the original PyFPDF lib with [web2py](http://www.web2py.com/) is described here: <https://github.com/reingart/pyfpdf/blob/master/docs/Web2Py.md>
 
 `v1.7.2` of PyFPDF is included in `web2py` since release `1.85.2`: <https://github.com/web2py/web2py/tree/master/gluon/contrib/fpdf>
+
+
+## FastAPI ##
+[FastAPI](https://fastapi.tiangolo.com/) is:
+> a modern, fast (high-performance), web framework for building APIs with Python 3.7+ based on standard Python type hints.
+
+The following code shows how to generate a PDF file via a POST endpoint that receives a JSON object. The JSON object can be used to write into the PDF file. The generated PDF file will be returned back to the user/frontend as the response. 
+
+
+```python
+from fastapi import FastAPI, Request, Response
+
+from fpdf import FPDF
+
+import warnings
+warnings.filterwarnings("ignore")
```

Comment:
Why disabling those warnings?
I think it may be better to remove those 2 lines, as we actually advise to enable warnings:
https://github.com/PyFPDF/fpdf2/blob/master/CHANGELOG.md#displaying-deprecation-warnings

Reply:
Yeah that makes sense. I'll remove those warnings in the next PR.

Reply:
You mean in the next commit, or you want to create another PR once this one is merged?

Reply:
Whoops my bad. Still not familiar with the Github lingo. Yes, I meant in the next commit. 

Honestly, I enjoy having discussions and making PRs. Even though this PR is not really that impressive as it is just documentation/guide. But, good starting experience nonetheless 👍

Reply:
> Whoops my bad. Still not familiar with the Github lingo. Yes, I meant in the next commit.

No worries, there is indeed a lot of lingo 😅

> Honestly, I enjoy having discussions and making PRs. Even though this PR is not really that impressive as it is just documentation/guide. But, good starting experience nonetheless 👍

I'm very glad that you do 😊 Thank you for this feedback.
And there is no small contribution. Documentation is often overlooked, but is essential,
and the time you are taking now to craft a good example will be valuable to many users
who will visit our this documentation page in the future!

Reply:
Thank you for your kind words. Truly means a lot. Much appreciated~!